### ISSUE TITLE
`pageconfig.json` has no effect - documentation typo?

### ISSUE THREAD

**Issue by achimgaedke** [jupyterlab/jupyterlab#11117]
In the jupyterlab documentation, I read about `pageconfig.json`
* https://jupyterlab.readthedocs.io/en/stable/user/extensions.html?highlight=pageconfig.json#enabling-and-disabling-extensions
* https://jupyterlab.readthedocs.io/en/stable/user/directories.html?highlight=pageconfig.json#labconfig-directories

These paragraphs read confusing and slightly contradictory.

Then I used `jupyter labextension disable` and see what it does... and it works on `labconfig/page_config.json`.
Renaming this file to `pageconfig.json` leads to all configurations in this file ignored.

In short, I believe this is a typo: `s/pageconfig.json/page_config.json/`

**Comment by krassowski** [comment]
I believe you are right. Would you like to submit a pull request correcting these typos? The convenience `Edit on GitHub` button in the documentations should now work again (we had issues with it before, but it was just fixed :tada:).
